doc/colvars-refman-main.tex

```diff
@@ -6325,6 +6325,63 @@ \subsection*{Overview}
   \simkey{widths}{\texttt{histogramGrid}}{lowerBoundaries}
 \end{itemize}
 
+\cvnamdonly{
+\cvsubsec{Reweighted free-energy landscapes from accelerated MD}{sec:reweighted_fel_from_aMD}
+
+The \texttt{reweightaMD} can be used to build reweighted free-energy landscapes of a set of collective variables on-the-fly from accelerated molecular dynamics (aMD)\cite{hamelberg2004} and Gaussian aMD (GaMD)\cite{Miao2015}. More specifically, it collects the boost potential, $\Delta V(\mathbf{X})$, and computes the corresponding free-energies:
+\begin{equation}\label{amd-reweighting}
+	A(\xi)=-\frac{1}{\beta}\ln\langle\exp\lbrace\beta\Delta V(\mathbf{X})\delta[\xi(\mathbf{X})-\xi]\rbrace\rangle.
```

Comment:
`\beta` is not introduced, and it may also be better to use explicitly kB T to target a broader audience.

Reply:
`\beta` is clarified to be `1/k_{B}T`.